Roslyn.slnx

```diff
@@ -0,0 +1,414 @@
+<Solution>
```

Comment:
Just curious, this slnx file was produced by `dotnet sln migrate` or some other mechanism?

Reply:
Yes, no manual changes appear to be needed. I will do a few more spot checks before merging though (verify count of projects before and after, verify numbers of tests running in CI.)